deployment/ansible/deploy.yml

```diff
@@ -56,7 +56,7 @@
           port: 3114
       chain:
         persist: true
-        db_path: "./db64"
+        db_path: "./db65"
```

Comment:
Is the mempool sorting part of consensus? But better bump one time too many!

Reply:
Mmm... I was thinking the mempool key. Though that table is not persisted so I could have avoided.

One more no issue though.